### PR TITLE
Feature exclude views non delta

### PR DESCRIPTION
**Pull request by souravg-db**
Added changes to exclude views from scan

### CODE REVIEW

File: discoverx/dx.py [hunk 1]
```diff
@@ -117,6 +117,7 @@ def scan(
         rules="*",
         sample_size=10000,
         what_if: bool = False,
+        data_source_formats: list[str] = ["DELTA"],
```
Comment (edurdevic) [inline discoverx/dx.py:120]: Could you please document this in the docstring along with the possible values?

**Review by edurdevic — APPROVED** [review]
LGTM!
Nice work @souravg-db !

File: discoverx/dx.py (hunk 1 above)
Comment (souravg-db) [reply discoverx/dx.py:120]: Done

**Review by edurdevic — APPROVED** [review]
LGTM.
This addresses #72

**Pull request merged**